test/test_gen_single.py

```diff
@@ -1,181 +1,211 @@
 import unittest
-import os
-import sys
-sys.path.insert(1, os.path.join(sys.path[0], '..'))
-
-from genson import Schema
 import base
 
+def check(self, instance, options, expected):  # alias for readability
```

Comment:
I would prefer not to do this. While converting everything to simple, positional arguments takes less space to write, it makes the tests harder to read, and code readability/maintainability is always my # 1 concern.